docker/dev_env/expand_aliases.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import sys
+from pathlib import Path
+
+
+SHARED_ALIASES = {
+    "j": ["just"],
+    "nuxt": ["j", "p", "frontend", "dev"],
```

Comment:
This feels like preferential treatment for the frontend 😄.

Reply:
It was just the first useful one I could think of that clearly illustrated multi-term expansion :sweat_smile: